index.html

```diff
@@ -4,30 +4,36 @@
   <title>Repository Hunter</title>
   <link rel="shortcut icon" href="http://d15dxvojnvxp1x.cloudfront.net/assets/favicon.ico">
   <link rel="icon" href="http://d15dxvojnvxp1x.cloudfront.net/assets/favicon.ico">
-  <link rel="stylesheet" type="text/css" media="all" href="css/styles.css">
+  <base href="./">
+  <link rel="stylesheet" type="text/css" media="all" href="css/style.css">
   <script type="text/javascript" src="js/jquery-1.10.2.min.js"></script>
 </head>
 <!-- to be updated soon -->
 <body>
   <div id="head">
         <h1>Repository Hunter</h1>
-        <p>Enter a Github username and slap the button to display profile information.</p>    
-        <input type="text" name="username" id="username" placeholder="{  / Github username / }">    
-        <a href="#" id="submit">Get Information</a>    
-        <div id="data" class="clearfix">
-        </div>
+        <form>
+          <label for="username">
+            Enter a Github username and slap the button to display profile information.
+          </label>    
+          <input type="text" name="username" id="username" placeholder="{  / Github username / }">    
+          <button type="submit" id="submit">Get Information</button>    
+        </form>
+        <div id="data" class="clearfix"></div>
   </div>
 <script type="text/javascript">
   $(function(){
+    var $username = $('#username');
+    var $data = $('#data');
+
     $('#submit').on('click', function(e){
       e.preventDefault();
-      $('#data').html('<div id="loader"><img src="css/loader.gif" alt="loading..."></div>');    
-      var username = $('#username').val();
+      var username = $username.val();
```

Comment:
<li> The loading gif needs to be restored as without it, things are not intuitive.